src/App.js

```diff
@@ -21,24 +22,29 @@ const App = () => {
     dispatch(zipDataFetch());
   }, [])
 
+  const [selectedCounty, setSelectedCounty] = useState('')
```

Comment:
Oh, just saw this. We will move it to redux when we do a data-related PR. Good for now!